### PR TITLE
btrfs: add pt_BR translation

### PR DESCRIPTION
**Pull request by isaacvicente**
This commit is a rewrite of an existing one I did, as I didn't follow the imperative mood of the style guide.
PS.: I already deleted the old PR in order to have this one.

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**
- `btrfs-progs v5.19.1`

### CODE REVIEW

File: pages.pt_BR/linux/btrfs.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+> Mais informações: <https://btrfs.wiki.kernel.org/index.php/Manpage/btrfs>.
+- Criar subvolume:
+
+`sudo btrfs subvolume create {{caminho/para/subvolume}}`
+
+- Liste subvolumes:
+
+`sudo btrfs subvolume list {{caminho/para/ponto_de_montagem}}`
+
+- Mostre informações de uso do espaço:
```
Comment (navarroaxel) [inline pages.pt_BR/linux/btrfs.md:14]: I don't think `subjuntivo` fits here
```suggestion
- Mostra informações de uso do espaço:
```

File: pages.pt_BR/linux/btrfs.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# btrfs
+
+> Um sistema de arquivos baseado no princípio copy-on-write (COW) para Linux.
+> Alguns subcomandos como `btrfs device` têm sua própria documentação de uso.
+> Mais informações: <https://btrfs.wiki.kernel.org/index.php/Manpage/btrfs>.
+- Criar subvolume:
+
+`sudo btrfs subvolume create {{caminho/para/subvolume}}`
+
+- Liste subvolumes:
```
Comment (navarroaxel) [inline pages.pt_BR/linux/btrfs.md:10]:
```suggestion
- Lista subvolumes:
```

File: pages.pt_BR/linux/btrfs.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+`sudo btrfs subvolume list {{caminho/para/ponto_de_montagem}}`
+
+- Mostre informações de uso do espaço:
+
+`sudo btrfs filesystem df {{caminho/para/ponto_de_montagem}}`
+
+- Ative a cota:
```
Comment (navarroaxel) [inline pages.pt_BR/linux/btrfs.md:18]:
```suggestion
- Ativa a cota:
```

File: pages.pt_BR/linux/btrfs.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+`sudo btrfs quota enable {{caminho/para/subvolume}}`
+
+- Mostre a cota:
```
Comment (navarroaxel) [inline pages.pt_BR/linux/btrfs.md:22]:
```suggestion
- Mostra a cota:
```

File: pages.pt_BR/linux/btrfs.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
+# btrfs
+
+> Um sistema de arquivos baseado no princípio copy-on-write (COW) para Linux.
+> Alguns subcomandos como `btrfs device` têm sua própria documentação de uso.
+> Mais informações: <https://btrfs.wiki.kernel.org/index.php/Manpage/btrfs>.
+- Criar subvolume:
```
Comment (navarroaxel) [inline pages.pt_BR/linux/btrfs.md:6]:
```suggestion

- Cria subvolume:
```

**Comment by isaacvicente** [comment]
It's possible to a maintainer create a new line above "- Crie um novo subvolume"

**Comment by isaacvicente** [comment]
I messed up some details. Recreating a new PR (again :p).

**Comment by navarroaxel** [comment]
@isaacvicente you can edit the branch and reopen the PR. there is no requirement to create a new PR.

**Comment by isaacvicente** [comment]
Sorry about closing and remaking the PR. Those PRs I did were my first ones, so I'm not used to the Github itself. I will study more to make my contributions a lot easier to maintainers review.

**Pull request closed without merging**